documentation/airport-guides/egll.md

```diff
@@ -0,0 +1,67 @@
+### About London Heathrow
+
+Heathrow Airport (Also known as London Heathrow) (ICAO: LKPR, IATA: LHR) is the main and biggest airport in the United Kingdom, acting as the primary hub for British Airways (ICAO: BAW, callsign: SPEEDBIRD), and the primary operating airport for Virgin Atlantic (ICAO: VIR, callsign: VIRGIN). 
+
+Heathrow Airport serves the majority of London air travel, as well as the majority of southern Britain. It is located 14 miles to the West of London's City Center. It is one of the six london airports servicing London (Along with City, Stansted, Luton, Southend and Gatwick). 
```

Comment:
change to `It is one of the 6 commercial airports which service London (along with City, Stansted, Luton, Southend and Gatwick).`